pages/linux/http-prompt.md

```diff
@@ -0,0 +1,15 @@
+# http-prompt
+
+> An interactive command-line HTTP client featuring autocomplete and syntax highlighting.
+
+- Launch a session targeting the default url of http://localhost:8000 or the previous session:
+
+`http-prompt`
+
+- Launch a session with a given url:
+
+`http-prompt {{http://example.com}}`
+
+- Launch a session with some initial options:
+
+`http-prompt {{localhost:8000/api}} --auth user:pass username={{user}}`
```

Comment:
Not sure, but perhaps everything from `--auth` should be in a `{{}}` token?

Reply:
I think `user:pass` is part of the syntax. You enter the username and then you are prompted for the password.

Reply:
Yes, the `--auth` is the option key, and `user:pass` is the option value.
I've simplified the documentation.